lib/template-compiler.js

```diff
@@ -21,12 +21,20 @@ ConditionalTemplateCompiler.prototype.transform = function(ast) {
 ConditionalTemplateCompiler.prototype.processParams = function(node) {
   var b = this.syntax.builders;
 
-  Object.keys(compileFlags).forEach(function(flag) {
+  for (var flag in compileFlags) {
```

Comment:
Nope!

In the really unlikely case that someone decides to do something silly with the feature flags themselves, the worst case scenario is a little bit of extra iteration that isn't required

`for( ... in ... )` is noticeably faster and so worth the boost in compile speed I think.